README.md

```diff
@@ -0,0 +1,38 @@
+# Provider
+Provider streams the iOS devices for controlfloor. This bridges the phone to the browser.
+
+# Basic Install Instructions
+* Clone repos
+git clone ios_remote_provider
+git clone controlfloor
+git clone ios_video_app
+
+* Build nng - https://github.com/nanomsg/nng
+cmake, make, make install
+
+* Build repos
+
+cd controlfloor
+make
+./main run
+
+Run https://yourip:8080 to see if controlfloor is running
+
+cd ios_remote_provider
+Edit config.json to add your Apple developer details
+make
+security unlock-keychain login.keychain #to make sure developer details are there for xcode build
+make wda
+./main register
```

Comment:
Mention should be made to "press enter to use the default password" as this tripped up a few users.